vertx-mongo-client/src/main/java/io/vertx/ext/mongo/package-info.java

```diff
@@ -364,6 +364,294 @@
  * {@link examples.Examples#example16_d1}
  * ----
  *
+ * == Storing/Retrieving files and binary data
+ *
+ * The client can store and retrieve files and binary data using MongoDB GridFS. The
+ * {@link io.vertx.ext.mongo.MongoGridFsClient} can be used to send a whole files or buffers
+ * to GridFS. If the object being stored is large, it can be stored one buffer at a time so as
```

Comment:
When the object being stored is large, it can be sent as a stream of chunks to keep the memory consumption low